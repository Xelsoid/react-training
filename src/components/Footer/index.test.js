import React from 'react';
import ComponentExample from './index';

describe('Component test example', () => {
  it('renders correctly', () => {
    const component = shallow(<ComponentExample />);
    expect(component).toMatchSnapshot();
  });
});
