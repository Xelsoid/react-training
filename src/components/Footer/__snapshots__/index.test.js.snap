// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test renders correctly 1`] = `
<div
  className="footer-wrapper"
>
  <footer
    className="footer"
  >
    <SiteTitle />
  </footer>
</div>
`;
